src/components/StructuredNavigation/NavUtils/SectionHeading.js

```diff
@@ -45,9 +46,17 @@ const SectionHeading = ({
     sectionRef,
     isCanvas: true,
     canvasDuration: duration,
-    setIsOpen
+    setSectionIsCollapsed
   });
 
+  // Collapse/Expand section when all sections are collapsed/expanded respectively
+  useEffect(() => {
+    // Do nothing for root structure items
+    if (!isRoot) {
+      isCollapsed ? setSectionIsCollapsed(true) : setSectionIsCollapsed(false);
+    }
```

Comment:
How about simplifying this?
```suggestion
    if (!isRoot) {
      setSectionIsCollapsed(isCollapsed);
    }
```